.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: php
 
+sudo: false
```

Comment:
Using containers with travis CI.

Reply:
Afair this is deprecated and will be removed

Reply:
Hmmm

Reply:
See https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures

Reply:
@staabm Thank you. I did not check this news. I understand.
@antonmedv I will send new PR later.